Automattic/pocket-casts-ios
# Playback 2023: entry point

| 📘 Part of: #1142 | 🎨 Designs: [Figma](https://www.figma.com/file/lH66LwxxgG8btQ8NrM0ldx/End-of-Year?type=design&node-id=1599-20385&mode=design) |
|:---:|:---:|

Updates the logic and visuals for the Playback 2023 modal, profile badge, and profile card.

| Modal | Badge | Card |
| ------ | ------ | ----- |
| ![Simulator Screenshot - iPhone 15 - 2023-10-16 at 13 20 43](https://github.com/Automattic/pocket-casts-ios/assets/7040243/6ef00ee4-b09e-497f-b05c-15ebe1107616) | ![Simulator Screenshot - iPhone 15 - 2023-10-16 at 13 19 18](https://github.com/Automattic/pocket-casts-ios/assets/7040243/4a8c4f5f-4557-4655-8129-839bd2741a6a) | ![Simulator Screenshot - iPhone 15 - 2023-10-16 at 13 19 26](https://github.com/Automattic/pocket-casts-ios/assets/7040243/a2ae75c4-3b77-459b-afbb-3ec99f0a3655) |

## To test

1. Make sure you're logged in to an account that has a few episodes listened
2. Go to Profile > Settings > Beta Features > enable `endOfYear`
3. Restart the app
4. ✅ A modal should be shown
5. ✅  Tapping on "View My 2023" shows the stories
6. Dismiss it
7. ✅ There should be a badge on the "Profile" tab
8. Tap it
9. ✅ The badge should disappear
10. ✅ Check that there's a card on Profile prompting you to see your 2023
11. Tap it
12. ✅ Playback 2023 stories should start

If you want to reset the modal being shown and badge, go to Profile > Settings > Developer > Reset modal/profile badge

Please test on a big device (iPhone 15 Pro Max, for example), a normal device (iPhone 15), and a smaller device (iPod touch). Also on light/dark mode.

## Checklist
